spec/lib/parser_spec.rb

```diff
@@ -0,0 +1,24 @@
+require './lib/parser'
+require './spec/spec_helper'
+
+describe Parser do
+  describe "#first_line" do
+    subject(:first_line) { parser.first_line }
+
+    context "when the file exist" do    
+      let(:parser) { described_class.new("./spec/fixtures/games_test.log")}
+      
+      it "should return the first line of the file" do
```

Comment:
Por convenção não é recomendado utilizar a palavra `should` nas asserções pois elas devem ser objetivas
https://www.betterspecs.org/#should